lms/djangoapps/course_blocks/transformers/library_content.py

```diff
@@ -123,8 +123,10 @@ def check_child_removal(block_key):
             Return True if selected block should be removed.
 
             Block is removed if it is part of library_content, but has
-            not been selected for current user.
+            not been selected for current user, with staff as an exemption.
             """
+            if usage_info.has_staff_access.has_access:
```

Comment:
i am interested in knowing object type of `has_staff_access` is an object of `AccessResponse` class?

Reply:
Yes, the output of _has_staff_access_ is _AccessResponse_ object, which in turn calls the has_access method of that object

Reply:
Ok, then `usage_info.has_staff_access` should work. Check the example. 

https://github.com/edx/edx-platform/blob/c4fc4b5df60c6eb6728e98f10c3dac57431cc3f6/lms/djangoapps/course_api/blocks/transformers/milestones.py#L69